great_expectations/self_check/util.py

```diff
@@ -159,7 +172,7 @@
         )
         try:
             getattr(sqla_bigquery, "INTEGER")
-            bigquery_types_tuple = {}
+            bigquery_types_tuple = {}  # type: ignore[var-annotated]
```

Comment:
Are there tradeoffs between adding this `ignore`s vs marking it as `Any`?

Reply:
I don't think so. But I generally try to avoid `Any`s if I can.

I tend to think of an `Any` as a more permanent, "we are giving up on narrowing this type".

Whereas this `ignore` is more of a "we just aren't dealing with this now but we might come back to it".

Honestly, the name of this variable and its usage were confusing.